### PR TITLE
Adds Navy Blue scrubs

### PR DESCRIPTION
**Pull request by redstryker**
Adds Navy Blue scrubs.  Also adds them to the loadout.

### CODE REVIEW

File: code/modules/clothing/under/jobs/medsci.dm [hunk 1]
```diff
@@ -204,6 +204,14 @@
 	worn_state = "scrubsblack"
 	rolled_sleeves = -1
 
+/obj/item/clothing/under/rank/medical/navyblue
+	name = "medical scrubs"
+	desc = "It's made of a special fiber that provides minor protection against biohazards. This one is in navy blue."
+	icon_state = "scrubsnavyblue"
+	item_state = "bl_suit"
```
Comment (Techhead0) [inline code/modules/clothing/under/jobs/medsci.dm]: Is `bl_suit` a typo? It's the same as the black scrubs.

**Comment by redstryker** [comment]
@Techhead0 , yeah, for some reason I put the 'l' there.  My bad.

**Comment by Hubblenaut** [comment]
No equivalent headpiece?

**Comment by redstryker** [comment]
When I get around to it, I will.  I honestly forgot about that.

**Pull request closed without merging**